### PR TITLE
`select-notifications` - Add filter for "Anything other than issues and PRs"

### PR DESCRIPTION
**Pull request by fregante**
- Closes https://github.com/refined-github/refined-github/issues/6980

I think "others" includes

- new tags
- comments on commits
- discussions
- deployments
- cancelled actions, if notifications are active
- dependabot alerts?

## Test URLs

https://github.com/notifications

## Screenshot

gif2

### CODE REVIEW

File: source/features/select-notifications.tsx [hunk 1]
```diff
@@ -85,6 +89,7 @@ function createDropdownList(category: Category, filters: Filter[]): JSX.Element
 		'Pull requests': <GitPullRequestIcon className="color-fg-muted"/>,
 		Issues: <IssueOpenedIcon className="color-fg-muted"/>,
 		Open: <CheckCircleIcon className="color-fg-success"/>,
+		Others: <SquirrelIcon className="color-fg-muted"/>,
```
Comment (fregante) [inline source/features/select-notifications.tsx:92]: https://primer.style/foundations/icons/squirrel-16

Alternatives: https://primer.style/foundations/icons#16px

Comment (fregante) [reply source/features/select-notifications.tsx:93]: Suggestions welcome even after the PR is merged

File: source/features/select-notifications.tsx [hunk 2]
```diff
@@ -13,22 +13,27 @@ import {
...
 	IssueOpenedIcon,
+	SquirrelIcon,
 	XCircleIcon,
 } from '@primer/octicons-react';
 
 import features from '../feature-manager.js';
 import observe from '../helpers/selector-observer.js';
 
+const prIcons = ':is(.octicon-git-pull-request, .octicon-git-pull-request-closed, .octicon-git-pull-request-draft, .octicon-git-merge)';
+const issueIcons = ':is(.octicon-issue-opened, .octicon-issue-closed, .octicon-skip)';
```
Comment (fregante) [inline source/features/select-notifications.tsx:24]: Also more comprehensive bugfix for:

- https://github.com/refined-github/refined-github/issues/6801
- https://github.com/refined-github/refined-github/pull/6803

File: source/features/select-notifications.tsx (hunk 1 above)
Comment (waldyrious) [reply source/features/select-notifications.tsx:93]: Some ideas:
- https://primer.style/foundations/icons/apps-16
- https://primer.style/foundations/icons/stack-16
- https://primer.style/foundations/icons/unread-16
- https://primer.style/foundations/icons/filter-remove-16
- https://primer.style/foundations/icons/rss-16

**Pull request merged**